lib/ReactViews/DragDropFile.tsx

```diff
@@ -105,7 +105,7 @@ class DragDropFile extends React.Component<PropsType> {
           [Styles.isActive]: this.props.viewState.isDraggingDroppingFile
         })}
       >
-        {isDefined(this.props.viewState.isDraggingDroppingFile) ? (
+        {this.props.viewState.isDraggingDroppingFile ? (
```

Comment:
This resulted in an invisible div below the main app all the time and the browser sometimes scrolls to reveal a white blank space. When `this.props.viewState.isDraggingDroppingFile` the div is correctly positioned at the center of the screen.